cluster/addons/rbac/cluster-loadbalancing/glbc/roles.yaml

```diff
@@ -63,3 +63,7 @@ rules:
 - apiGroups: ["networking.k8s.io"]
   resources: ["ingressclasses"]
   verbs: ["get", "list", "watch", "update", "create", "patch"]
+# GLBC uses leases for leader election
+- apiGroups: ["coordination.k8s.io"]
+  resources: ["leases"]
+  verbs: ["get","list","watch","update",'"create","patch"]
```

Comment:
can this not be restricted more?

Reply:
it seems there is a TODO there you may want to check in addition to this
https://github.com/kubernetes/ingress-gce/blob/37a3586364079f4af0726a0fc444135e42a56bf7/cmd/glbc/main.go#L226-L228

@wojtek-t @freehan

Reply:
for the implementation it seems that only get, create and update are needed

https://github.com/kubernetes/ingress-gce/blob/37a3586364079f4af0726a0fc444135e42a56bf7/vendor/k8s.io/client-go/tools/leaderelection/resourcelock/leaselock.go

Reply:
Thanks! I did not realize we were using an intermediate step. I will followup about that on the ingress-gce repo so that we have migrated to the current version. Created https://github.com/kubernetes/ingress-gce/issues/1713 to track that change.

I also updated the permissions to only request `get`, `create`, and `update`

Reply:
> it seems there is a TODO there you may want to check in addition to this
https://github.com/kubernetes/ingress-gce/blob/37a3586364079f4af0726a0fc444135e42a56bf7/cmd/glbc/main.go#L226-L228

We just migrated to leases this year. It needs a bit more time.